src/bin/vip-wp.js

```diff
@@ -72,6 +112,66 @@ commandWrapper( {
 
 		let result;
 		let rl;
+		let subShellRl;
+
+		if ( isSubShell ) {
+			console.log( `Entering WP-CLI subshell mode for ${ formatEnvironment( envName ) } on ${ appName } (${ appId })` );
+
+			subShellRl = readline.createInterface( {
+				input: process.stdin,
+				output: process.stdout,
+				terminal: true,
+				prompt: `${ appName }.${ envName }> `,
+				// TODO make history persistent across sessions for same env
+				historySize: 200,
+			} );
+
+			subShellRl.on( 'line', async line => {
+				console.log( 'received line:', line );
```

Comment:
We'll need to remove this line (and any related debug code) before shipping.

Reply:
whoops, this was added to debug but forgot it there. Should be fixed by this https://github.com/Automattic/vip/pull/280/commits/e0fa601f73e0a0474fbcc306546baa157e71718e